oolab/src/main/java/agh/ics/oop/model/MapDirection.java

```diff
@@ -0,0 +1,37 @@
+package agh.ics.oop.model;
+
+public enum MapDirection{
+    NORTH,
+    EAST,
+    SOUTH,
+    WEST;
+
+    @Override
+    public String toString(){
+        return switch (this) {
+            case NORTH -> "Północ";
+            case EAST -> "Wschód";
+            case SOUTH -> "Południe";
+            case WEST -> "Zachód";
+        };
+    }
+
+    public MapDirection next(){
+        int nextDirectionIndex = (this.ordinal() + 1) % values().length;
```

Comment:
sprytnie :D ale może być problematyczne, bo wystarczy że ktoś przestawi jakąś stałą wyżej, albo doda nową i nie popatrzy. To jest wbrew pozorom dość powszechny problem, np. przy zapisywaniu enumów do baz danych też często polega się na ich kolejności i jest to bardzo ryzykowne.